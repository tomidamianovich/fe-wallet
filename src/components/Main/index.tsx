import React from 'react';
import { CurrencyType, RateType } from '../../utils/type';
import './styles/index.scss';

type Props = {
  currencies: CurrencyType[],
  rates: RateType[]
};

const Main: React.FC<Props> = ({
    currencies, 
    rates
  }) => 
    <main className="fe-wallet__main">
      <p> Currencies Amount: {currencies.length}</p>
      <p> Rates Amount: {rates.length}</p>
    </main>
;

export default Main;