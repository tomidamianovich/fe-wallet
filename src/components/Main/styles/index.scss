.fe-wallet__content__main {
  
}