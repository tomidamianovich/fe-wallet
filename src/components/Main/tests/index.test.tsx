import { render } from '@testing-library/react';
import Main from '../index';

test('renders Main main without items', () => {
  const { asFragment } = render(<Main currencies={[]} rates={[]} balances={[]} />);
  expect(asFragment()).toMatchSnapshot();
});
