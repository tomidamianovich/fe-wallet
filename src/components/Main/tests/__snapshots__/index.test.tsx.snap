// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Navigation main with items 1`] = `
<DocumentFragment>
  <main
    class="fe-wallet__main"
  />
</DocumentFragment>
`;

exports[`renders Navigation main without items 1`] = `
<DocumentFragment>
  <main
    class="fe-wallet__main"
  />
</DocumentFragment>
`;
